app/helpers/metric_chart_helper.rb

```diff
@@ -0,0 +1,51 @@
+module MetricChartHelper
+  def calculate_transaction_difference(current_metrics, previous_metrics, total_label)
+    @current_metrics = current_metrics
+    @previous_metrics = previous_metrics
+    if increased?
+      "#{@current_metrics.sum - @previous_metrics.sum} (#{difference_increase_percentage} %) more #{total_label.gsub('Total transactions', 'transactions').downcase} than previous #{@previous_metrics.count} months".html_safe
```

Comment:
It might look better to use ```metric_to_human``` on the ```#{@current_metrics.sum - @previous_metrics.sum}``` and remove the space in the ```(#{difference_increase_percentage} %)```

Reply:
Hi @rossjones,

Had a problem with this in that the `metric_to_human` method makes it a data attribute which messes up the styling.

<img width="333" alt="metric_to_human" src="https://user-images.githubusercontent.com/5963488/37598576-7c96759c-2b7a-11e8-8c61-1501ecae2bd9.png">

I instead added a method `humanize_metric` which just does the conversion. Let me know what you think.